app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeActivity.kt

```diff
@@ -335,7 +335,7 @@ class ComposeActivity :
                 maximumTootCharacters = instanceData.maxChars
                 charactersReservedPerUrl = instanceData.charactersReservedPerUrl
                 updateVisibleCharactersLeft()
-                binding.composeScheduleButton.visible(instanceData.supportsScheduled)
+                binding.composeScheduleButton.visible(true)
```

Comment:
I think this line is entirely unnecessary as the button is visible after layout inflation (if not, change it in the layout)

Reply:
Ah, I found the button - lets see if it stays visible when I remove the line

Reply:
All looks ok - the button is visible